pages/common/xml-depyx.md

```diff
@@ -0,0 +1,16 @@
+# xml-depyx
```

Comment:
The title should be the command as it's invoked:
```suggestion
# xml depyx
```

Reply:
Resolved all 13 places:
xml.md:# xml
xml-canonic.md:# xml canonic
xml-depyx.md:# xml depyx
xml-edit.md:# xml edit
xml-elements.md:# xml elements
xml-escape.md:# xml escape
xml-format.md:# xml format
xml-list.md:# xml list
xml-pyx.md:# xml pyx
xml-select.md:# xml select
xml-transform.md:# xml transform
xml-unescape.md:# xml unescape
xml-validate.md:# xml validate